glean_parser/parser.py

```diff
@@ -274,6 +276,7 @@ def parse_objects(filepaths, config={}):
     :param config: A dictionary of options that change parsing behavior.
         Supported keys are:
         - `allow_reserved`: Allow values reserved for internal Glean use.
+        - `do_not_disable_expired`: Don't mark expired metrics as disabled.
```

Comment:
nit: can please we add the summary of "what's this is useful for" that was added to the `HISTORY.rst` file?